src/containers/actions/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style */
+
```

Comment:
## Move the action files
- [ ] The `action.js` module needs to be moved to the `src` directory. You only need to have the `bookForm.js` and the `bookList.js` components in this directory.